ideas/151-ens-usernames.md

```diff
@@ -0,0 +1,66 @@
+# Username ENS subdomain Registrar
+
+## Preamble
+
+    Idea: 151-ens-usernames
+    Title: Username ENS subdomain Registrar
+    Status: Draft
+    Created: 2017-11-20
+    Replaces: 027-ens-usernames
+
+## Summary
+
+Eliminates the need to copy/scan - and worse, type - long hexadecimal addresses / public keys, by providing an [ENS](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-137.md) subdomain registry and recognition of ENS names in Status for interacting with other people in Status.
+
+## Swarm Participants
```

Comment:
What other roles do we need here? At what point do you think it'd be useful to add Clojure contributors.

Probably also want PM here, cc @chadyj

Reply:
after discussion on slack: since this is fairly self-contained it'd be useful to have clojure dev here